tests/FakeItEasy.IntegrationTests/FakeItEasy.IntegrationTests.csproj

```diff
@@ -6,6 +6,10 @@
     <SignAssembly>true</SignAssembly>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(TargetFramework)' == 'net5.0'">
+    <DefineConstants>$(DefineConstants);FEATURE_STRING_CONTAINS_COMPARISONTYPE</DefineConstants>

```

Comment:
A perfectly good change. Have we reached the point where we should swap to indicate that `String.Contains` lacks comparison type? We can wait until the next supported TFM change.

Reply:
> Have we reached the point where we should swap to indicate that `String.Contains` lacks comparison type?

In this project, yes (2 TFMs have the feature, 1 doesn't)
But in FakeItEasy.csproj, we have 2 TFMs with the feature and 2 without.

I guess we can swap now.

Reply:
For me, it's more about expected future work. I like the idea of new TFMs having cleaner `.csproj`s. Make the old TFMs take more work! But it's not a huge deal. Whether or when we change doesn't rile me up too much.

Reply:
> I like the idea of new TFMs having cleaner `.csproj`s.

Me too. Let's do it.